nanshe_ipython.ipynb

```diff
@@ -1151,7 +1152,7 @@
         "all_tr_shape_srcs = ColumnDataSource(data=dict(traces_shape=traces.shape))\n",
         "all_tr_srcs = ColumnDataSource(data=dict(\n",
         "    traces=numpy.frombuffer(\n",
-        "        traces.tobytes(),\n",
+        "        zlib.compress(traces.tobytes()),\n",
```

Comment:
Compress the `byte`-string representation of the traces using `zlib`. Then store the result back as a NumPy `uint8` array for efficient `base64` encoding and easy use with pako's decompression code.